components/bl-transferList-component/src/transfer-buttons.js

```diff
@@ -0,0 +1,136 @@
+export const TransferButtons = props => {
+  const {
+    enableMoveAll,
+    allSelected,
+    left,
+    right,
+    leftSelected,
+    rightSelected,
+    setAllSelected,
+    setLeft,
+    setRight,
+    eventHandlers,
+  } = props;
+
+  const { onChange } = eventHandlers;
+
+  const isMoveAllToRightDisabled = !left.length;
+  const isMoveAllToLeftDisabled = !right.length;
+  const isMoveSelectedToRightDisabled = !leftSelected.length;
+  const isMoveSelectedToLeftDisabled = !rightSelected.length;
```

Comment:
17-20 - as it was used only one time, and logic is not very complicated, I think we could write it without external variables. So, that's could be moved to props

```
<button
  onClick={ moveSelectedToRight }
  disabled={ !leftSelected.length }
  className="control-buttons__item">
  &gt;
</button>
```

And so on...